experimental/packages/opentelemetry-sdk-metrics/src/index.ts

```diff
@@ -14,19 +14,74 @@
  * limitations under the License.
  */
 
-export { Sum, LastValue, Histogram } from './aggregator/types';
-export * from './export/AggregationTemporality';
-export * from './export/MetricData';
-export * from './export/MetricExporter';
-export * from './export/MetricProducer';
-export * from './export/MetricReader';
-export * from './export/PeriodicExportingMetricReader';
-export * from './export/InMemoryMetricExporter';
-export * from './export/ConsoleMetricExporter';
-export { InstrumentDescriptor, InstrumentType } from './InstrumentDescriptor';
-export * from './Meter';
-export * from './MeterProvider';
-export * from './ObservableResult';
-export { TimeoutError } from './utils';
```

Comment:
I believe this is intended for people to check error brands.

Reply:
Ah, yes. You're right. Added back in [e2400ce](https://github.com/open-telemetry/opentelemetry-js/pull/3197/commits/e2400cea78ae1502c17d1de1565bf50ad364f1c6). Thanks! :slightly_smiling_face: